.github/workflows/ci.yaml

```
@@ -66,18 +70,17 @@ jobs:
       run: |
         micromamba install openmm -c conda-forge
 
-    - name: Uninstall OpenMM
-      if: ${{ matrix.openmm == false && matrix.openeye == true }}
+    - name: Install Python <3.13 dependencies
+      if: ${{ matrix.python-version != '3.13' }}
       run: |
-        micromamba remove openmm mdtraj
-        # Removing mBuild also removes some leaves, need to re-install them
-        micromamba install rdkit packmol "lammps >=2023.08.02"
-
-    - name: Install AmberTools and RDKit
-      if: ${{ matrix.openeye == false }}
-      # Unclear why, but around October 2023 this downgrades JAX to broken 0.1.x builds
-      # and also uninstalls RDKit
-      run: micromamba install rdkit "ambertools =23" "lammps >=2023.08.02" "jax >=0.3" "jaxlib >=0.3" -c conda-forge
+        micromamba install \
+          mbuild \
+          foyer \
+          "openff-nagl ~=0.5" \
+          "lammps >=2023.08.02" \
+          "openmpi =4" \
+          smirnoff-plugins \
+          de-forcefields
```

(not blocking) Might be good to copy this comment from test_env.yaml
```suggestion
        # openmpi=4 is a shim for a build issue that's probably localized to conda-forge
        # https://github.com/conda-forge/lammps-feedstock/issues/207
        micromamba install \
          mbuild \
          foyer \
          "openff-nagl ~=0.5" \
          "lammps >=2023.08.02" \
          "openmpi =4" \
          smirnoff-plugins \
          de-forcefields
```